### ISSUE TITLE
launch kubevela failed for CRD not found

### ISSUE THREAD

**Issue by captainroy-hy** [oam-dev/kubevela#1130]
Since `ServiceMonitor.monitoring.coreos.com` is already removed from kubevela chart, when launch kubevela,  controller-manager crashed for metrics controller depends on watching `servicemonitors`.
```
2021-03-03T20:11:20.735+0900	INFO	controller	Starting Controller	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "ApplicationConfiguration", "controller": "oam/applicationconfiguration.core.oam.dev"}
I0303 20:11:21.680725   19627 request.go:621] Throttling request took 1.0421498s, request: GET:https://127.0.0.1:36763/apis/apiregistration.k8s.io/v1beta1?timeout=32s
2021-03-03T20:11:22.982+0900	ERROR	controller-runtime.source	if kind is a CRD, it should be installed before calling Start	{"kind": "ServiceMonitor.monitoring.coreos.com", "error": "no matches for kind \"ServiceMonitor\" in version \"monitoring.coreos.com/v1\""}
github.com/go-logr/zapr.(*zapLogger).Error
	/home/roywang/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128
sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start
	/home/roywang/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/source/source.go:117
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/home/roywang/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/internal/controller/controller.go:140
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start
	/home/roywang/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/internal/controller/controller.go:175
sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startRunnable.func1
	/home/roywang/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/manager/internal.go:661
2021-03-03T20:11:22.982+0900	INFO	controller	Starting workers	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "ManualScalerTrait", "controller": "oam/manualscalertrait", "worker count": 1}
2021-03-03T20:11:22.982+0900	INFO	controller	Stopping workers	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "ManualScalerTrait", "controller": "oam/manualscalertrait"}
2021-03-03T20:11:22.982+0900	INFO	controller	Starting Controller	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "ApplicationDeployment", "controller": "applicationdeployment"}
2021-03-03T20:11:22.982+0900	INFO	controller	Starting workers	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "HealthScope", "controller": "oam/healthscope.core.oam.dev", "worker count": 1}
2021-03-03T20:11:22.982+0900	INFO	controller	Stopping workers	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "HealthScope", "controller": "oam/healthscope.core.oam.dev"}2021-03-03T20:11:20.735+0900	INFO	controller	Starting Controller	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "ApplicationConfiguration", "controller": "oam/applicationconfiguration.core.oam.dev"}
I0303 20:11:21.680725   19627 request.go:621] Throttling request took 1.0421498s, request: GET:https://127.0.0.1:36763/apis/apiregistration.k8s.io/v1beta1?timeout=32s
2021-03-03T20:11:22.982+0900	ERROR	controller-runtime.source	if kind is a CRD, it should be installed before calling Start	{"kind": "ServiceMonitor.monitoring.coreos.com", "error": "no matches for kind \"ServiceMonitor\" in version \"monitoring.coreos.com/v1\""}
github.com/go-logr/zapr.(*zapLogger).Error
	/home/roywang/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128
sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start
	/home/roywang/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/source/source.go:117
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/home/roywang/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/internal/controller/controller.go:140
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start
	/home/roywang/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/internal/controller/controller.go:175
sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startRunnable.func1
	/home/roywang/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/manager/internal.go:661
2021-03-03T20:11:22.982+0900	INFO	controller	Starting workers	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "ManualScalerTrait", "controller": "oam/manualscalertrait", "worker count": 1}
2021-03-03T20:11:22.982+0900	INFO	controller	Stopping workers	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "ManualScalerTrait", "controller": "oam/manualscalertrait"}
2021-03-03T20:11:22.982+0900	INFO	controller	Starting Controller	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "ApplicationDeployment", "controller": "applicationdeployment"}
2021-03-03T20:11:22.982+0900	INFO	controller	Starting workers	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "HealthScope", "controller": "oam/healthscope.core.oam.dev", "worker count": 1}
2021-03-03T20:11:22.982+0900	INFO	controller	Stopping workers	{"reconcilerGroup": "core.oam.dev", "reconcilerKind": "HealthScope", "controller": "oam/healthscope.core.oam.dev"}
```

Shall we add the CRD back to the chart?

**Comment by wonderflow** [comment]
It will be fixed by #1080 , we can remove this controller in first priority. Ping @WangYikewyk

**Comment by wonderflow** [comment]
For anyone has the same issue, a quick fix is to install the CRD here: https://github.com/oam-dev/kubevela/blob/master/pkg/controller/standard.oam.dev/v1alpha1/testdata/crds/servicemonitors.yaml
